@clalancette CI builds fails with error messages
```
-- Build files have been written to: /home/jenkins-agent/workspace/ci_linux/ws/build/mcap_vendor
[ 16%] [32mBuilding CXX object CMakeFiles/mcap.dir/src/main.cpp.o[0m
In file included from [01m[K/home/jenkins-agent/workspace/ci_linux/ws/build/mcap_vendor/_deps/mcap-src/cpp/mcap/include/mcap/reader.hpp:708[m[K,
                 from [01m[K/home/jenkins-agent/workspace/ci_linux/ws/build/mcap_vendor/_deps/mcap-src/cpp/mcap/include/mcap/mcap.hpp:3[m[K,
                 from [01m[K/home/jenkins-agent/workspace/ci_linux/ws/src/ros-tooling/rosbag2_storage_mcap/mcap_vendor/src/main.cpp:16[m[K:
[01m[K/home/jenkins-agent/workspace/ci_linux/ws/build/mcap_vendor/_deps/mcap-src/cpp/mcap/include/mcap/reader.inl:8:10:[m[K [01;31m[Kfatal error: [m[Kzstd_errors.h: No such file or directory
    8 | #include [01;31m[K<zstd_errors.h>[m[K
      |          [01;31m[K^~~~~~~~~~~~~~~[m[K
compilation terminated.
gmake[2]: *** [CMakeFiles/mcap.dir/build.make:76: CMakeFiles/mcap.dir/src/main.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:137: CMakeFiles/mcap.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
```
It seems some problem with finding header files from `zstd` library. In particular `zstd_errors.h`
Although I can build it from scratch with no errors on my local machine.

--- a/mcap_vendor/CMakeLists.txt
+++ b/mcap_vendor/CMakeLists.txt
@@ -50,7 +50,7 @@ macro(build_mcap_vendor)
     "$<BUILD_INTERFACE:${_mcap_include_dir}>"
     "$<INSTALL_INTERFACE:include>"
   )
-  target_link_libraries(mcap PRIVATE zstd::zstd)
+  ament_target_dependencies(mcap zstd)
 
   install(
     DIRECTORY
@@ -66,6 +66,8 @@ endmacro()
 build_mcap_vendor()
 
 ament_export_targets(export_mcap HAS_LIBRARY_TARGET)
+
+ament_export_dependencies(zstd_vendor zstd)
 
 ## Tests
 if(BUILD_TESTING)

